internal/database/migration/cliutil/drift_schema.go

```diff
@@ -0,0 +1,58 @@
+package cliutil
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"regexp"
+	"strings"
+
+	descriptions "github.com/sourcegraph/sourcegraph/internal/database/migration/schemas"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+type ExpectedSchemaFactory func(repoName, version string) (descriptions.SchemaDescription, bool, error)
```

Comment:
```suggestion
type ExpectedSchemaFactory func(filename, version string) (descriptions.SchemaDescription, bool, error)
```

(this arg is referred to as filename in all the implementations below)